deployments/prod/environment.py

```diff
@@ -86,4 +86,9 @@ def env() -> Mapping[str, Optional[str]]:
         # FIXME: Unset once the client ID becomes optional
         #        https://github.com/DataBiosphere/azul/issues/3088
         'AZUL_GOOGLE_OAUTH2_CLIENT_ID': '473200283737-h5e1l7neunbuesrtgjf8b12lb7o3jf1m.apps.googleusercontent.com',
+
+        'AZUL_INDEXER_CONCURRENCY': '300',
+
+        # $12.00/hour
+        'AZUL_BIGQUERY_RESERVED_SLOTS': '300',
```

Comment:
The number of slots should be derived programmatically from the concurrency of the `contribute` lambda (not the `-retry` one).